crates/conformance-test-utils/Cargo.toml

```diff
@@ -6,13 +6,18 @@ description = { workspace = true }
 homepage = { workspace = true }
 
 [dependencies]
-starknet = { workspace = true }
+ethers = { workspace = true }
+eyre = { workspace = true }
+kakarot_rpc_core = { path = "../core" }
+starknet_api = { workspace = true }
+reth-primitives = { workspace = true }
 serde = { workspace = true }
 serde_json = { workspace = true }
 serde_with = { workspace = true }
-kakarot_rpc_core = { path = "../core" }
-url = { workspace = true }
+starknet = { workspace = true }
 tokio = { workspace = true }
+url = { workspace = true }
 
 [dev-dependencies]
 cargo-husky = { workspace = true }
+katana-core = { workspace = true }
```

Comment:
no new line at the end of file, for vscode, set 

```
"files.insertFinalNewline": true
```

in config to avoid these errors!